django/db/backends/base/operations.py

```diff
@@ -177,16 +177,26 @@ def force_no_ordering(self):
         """
         return []
 
-    def for_update_sql(self, nowait=False, skip_locked=False):
+    def for_update_sql(self, nowait=False, skip_locked=False, of=[]):
```

Comment:
@orf we don't have [flake8-mutable](https://pypi.python.org/pypi/flake8-mutable) installed on CI.